cmd/influxd/server_integration_test.go

```diff
@@ -1469,7 +1469,7 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 		{
 			name:     `show continuous queries`,
 			query:    `SHOW CONTINUOUS QUERIES`,
-			expected: `{"results":[{"series":[{"name":"%DB%","columns":["name","query"],"values":[["my.query","CREATE CONTINUOUS QUERY \"my.query\" ON %DB% BEGIN SELECT count(value) INTO \"%DB%\".\"%RP%\".measure1 FROM \"%DB%\".\"%RP%\".myseries GROUP BY time(10m) END"]]}]}]}`,
+			expected: `{"results":[{"series":[{"name":"%DB%","columns":["name","query", "running"],"values":[["my.query","CREATE CONTINUOUS QUERY \"my.query\" ON %DB% BEGIN SELECT count(value) INTO \"%DB%\".\"%RP%\".measure1 FROM \"%DB%\".\"%RP%\".myseries GROUP BY time(10m) END",false]]}]}]}`,
```

Comment:
OK, I see what you're saying (actually I think you mean only against the data node that is running the query). I can see something like this being useful, as how else would I know if a CQ is hung somewhere on a node?

That said, I'm open for a more comprehensive approach to CQ management, and can delete the commit that shows CQ status. It's still doesn't show `lastRun` time, which we'd need as well.